### PR TITLE
Serverless pattern eventbridge schedule remove one time schedules cdk python

### PR DESCRIPTION
**Pull request by manikandanks2000**
*Issue #, if available:*

*Description of changes:*
Remove completed Amazon EventBridge Schedules and sent notification using Amazon SNS 

This pattern will create an Amazon EventBridge Scheduler and will run every five minutes and will use AWS Lambda  to identify expired and completed Amazon Evenbridge Scheduler tasks that are expired 7 days or more and will send an SNS notification once the activity is completed. The pattern is deployed using the AWS Cloud Development Kit (AWS CDK) for Python. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by pputhran — COMMENTED** [review]
This is missing some of the required files. Can you please check? Also can you expand how is this different from [Automatically delete schedules upon completion with Amazon EventBridge Scheduler](https://aws.amazon.com/blogs/compute/automatically-delete-schedules-upon-completion-with-amazon-eventbridge-scheduler/)

**Comment by manikandanks2000** [comment]
pattern file and project files updated and included in the merge request . thanks
blog referenced is providing steps on how to remove one time schedule using aws console and did not provide a CDK code for reference.

File: eventbridge-schedule-remove-one-time-schedules-cdk-python/example-pattern.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    "text": [
+      "Delete the stack: <code>cdk delete</code>."
+    ]
+  },
+  "authors": [
+    {
+      "name": "Manikandan Karimanal",
+      "image": "https://drive.google.com/file/d/16r4jGGMpPhu4Igez-lzstIe-L5nJVMxf/view",
+      "bio": "Solution Architect @AWS",
+      "linkedin": "https://www.linkedin.com/in/manikandanks/"
```
Comment (pputhran) [inline eventbridge-schedule-remove-one-time-schedules-cdk-python/example-pattern.json:55]: Can you please take a look at the error and fix the LinkedIn Id?

```
  '1. `authors[0].linkedin`: authors[0].linkedin. Please remove the URL in this property, only include the ID to your LinkedIn profile.\n'
  ```

**Review by pputhran — CHANGES_REQUESTED** [review]
linkedin should include just the ID not uri

File: eventbridge-schedule-remove-one-time-schedules-cdk-python/example-pattern.json (hunk 1 above)
Comment (manikandanks2000) [reply eventbridge-schedule-remove-one-time-schedules-cdk-python/example-pattern.json]: thanks for the input . fixed the example pattern file with linked id only.

File: eventbridge-schedule-remove-one-time-schedules-cdk-python/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+    ``` 
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+2. Change directory to the pattern directory:
+    ```
+    cd eventbridge-schedule-remove-one-time-schedule-to-sns-cdk-python
```
Comment (pputhran) [inline eventbridge-schedule-remove-one-time-schedules-cdk-python/README.md:25]: should this be `eventbridge-schedule-remove-one-time-schedules-cdk-python` ?

File: eventbridge-schedule-remove-one-time-schedules-cdk-python/README.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    ```
+    aws sns subscribe --topic-arn YOURSNSTOPICARN --protocol email --notification-endpoint YOUREMAIL
+    ```
+## How it works
+
+An Amazon EventBridge Scheduler job runs every five minutes and will use AWS lambda to identify and delete one time Amazon EventBridge Scheduler jobs expired more than 7 days.Scheduler job also sends a SNS notifications to confirm the activity is completed.
+Along with a schedule and topic, the CDK stack creates an IAM role and policy for Amazon EventBridge Scheduler to assume and send messages. 
+
+## Testing
+After the stack has been deployed, you can verify Amazon EventBridge is successfully publishing to the topic by viewing the topics "NumberOfMessagesPublished" metric in CloudWatch and verifying positive data points.
```
Comment (pputhran) [inline eventbridge-schedule-remove-one-time-schedules-cdk-python/README.md:49]: can you add any screenshot here to make it clear?

**Review by pputhran — CHANGES_REQUESTED** [review]
requesting a few changes

**Comment by manikandanks2000** [comment]
agreed. updated the folder names as advised.
architecture reference diagram available in img folder as below
/img/onetimeschedule-architecturediagram.png 
let me know if this should be placed in the parent folder or otherwise.
thanks again

**Comment by pputhran** [comment]
@manikandanks2000 I had to fix a couple of small changes in the example-pattern.json file. Please have a look and see if this applies to your other submissions as well. Rest looks good and this PR will be merged in soon. Thanks for your submission.

**Comment by manikandanks2000** [comment]
thanks for your help and the changes made.

**Comment by manikandanks2000** [comment]
@pputhran  i see the project merged successfully. but the github project folder name need to be corrected as below.   Let me know whether a new merge request is required to make this change. 

https://serverlessland.com/patterns/eventbridge-schedule-remove-one-time-schedules-cdk-python

Incorrect path used :
https://github.com/aws-samples/serverless-patterns/tree/main/eventbridge-schedule-remove-onetime-schedule-to-sns-cdk-python

Correct path to be used :
https://github.com/aws-samples/serverless-patterns/tree/main/eventbridge-schedule-remove-one-time-schedules-cdk-python

**Comment by pputhran** [comment]
Please open a new PR with the fix.

**Pull request merged**